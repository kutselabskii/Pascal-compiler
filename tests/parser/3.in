program ABC;
const a = 5;
begin
end
.