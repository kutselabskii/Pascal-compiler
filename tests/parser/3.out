|__ program
    |-- abc //Program_heading
    |__ //Block
        |-- //Declaration part
        |   |-- const
        |   |   |-- a
        |   |   |   |__ 5
        |__ //Statement part
            |__ //Statement sequence
